opentreemap/treemap/lib/map_feature.py

```diff
@@ -358,3 +359,64 @@ def _add_share_context(context, request, photos):
         'description': description,
         'image': photo_url,
     }
+
+
+def set_map_feature_updated_at():
+    cursor = connection.cursor()
+
+    # For a baseline, pull the most recent change to the MapFeature itself.
+    # This depends on the fact that all the MapFeature subclasses share the
+    # same id pool and ids do not overlap.
+    # NOTE: This MUST be run first. Additional update queries compare
+    # dates against the updated_at values set by this statement.
+    cursor.execute("""
+UPDATE treemap_mapfeature
+SET updated_at = a.updated_at
+FROM (
+  SELECT model_id as id, MAX(created) AS updated_at
+  FROM treemap_audit
+  WHERE treemap_audit.model IN ('Plot', 'RainBarrelLA', 'RainGardenLA',
+                                'TurfConcreteGardenLA')
```

Comment:
For a bit of future-proofing, consider replacing this hardcoded list of model names with a dynamically generated one by calling `[Model.map_feature_type for Model in leaf_subclasses(MapFeature)]`